cg/cli/workflow/commands.py

```diff
@@ -383,3 +384,22 @@ def microsalt_past_run_dirs(
     context.invoke(
         past_run_dirs, skip_confirmation=skip_confirmation, dry_run=dry_run, before_str=before_str
     )
+
+
+@click.command("tower-past-run-dirs")
+@SKIP_CONFIRMATION
+@ARGUMENT_BEFORE_STR
+@click.option(
+    "-w",
+    "--workflow",
+    "workflow",
+    help="Only cleans analyses from the specified workflow, if provided.",
+)
+@click.pass_context
```

Comment:
Is providing the workflow really an option? To me it seems necessary for the command to run at all. I would use click.argument instead

Reply:
Good catch! Thought it would be an option at first but I noticed that it was needed to instantiate the analysis api. Forgot to change it though!